pkg/apis/minio.min.io/v2/types.go

```diff
@@ -280,6 +280,11 @@ type TenantSpec struct {
 	Users []*corev1.LocalObjectReference `json:"users,omitempty"`
 	// *Optional* +
 	//
+	// Create buckets when creating a new tenant. Skip if bucket with given name already exists
+	// +optional
+	Buckets []string `json:"buckets,omitempty"`
```

Comment:
To make this future proof, I'd recommend to make this an array of bucket objects, as buckets have multiple properties (versioning, object-locking, quota, etc...) we can start with just plain buckets, but let's add the object so that we don't have to change this later in a breaking way

Reply:
The `MakeBucketOptions` supports `Region` and `ObjectLocking` which are added in this PR.